### PR TITLE
Move karaf features to features directory. Remove poms subdir.

### PR DESCRIPTION
**Pull request by davidgraeff**
Signed-off-by: David Graeff <david.graeff@web.de>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,23 +1,771 @@
...
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.openhab.addons.tmp</groupId>
-  <artifactId>org.openhab.addons.tmp</artifactId>
+  <groupId>org.openhab.addons</groupId>
+  <artifactId>org.openhab.addons.reactor</artifactId>
   <version>2.5.0-SNAPSHOT</version>
+
   <packaging>pom</packaging>
-  <name>openHAB 2 Add-ons</name>
```
Comment (J-N-K) [inline pom.xml]: Why did you change from "OpenHAB 2 Add-ons" to "OpenHAB Add-Ons"?

File: pom.xml [hunk 2]
```diff
@@ -1,23 +1,771 @@
...
+        <executions>
+          <execution>
+            <id>directories</id>
+            <goals>
+              <goal>directory-of</goal>
+            </goals>
+            <phase>initialize</phase>
+            <configuration>
+              <property>basedirRoot</property>
+			  <project>
```
Comment (J-N-K) [inline pom.xml]: wrong indentation

File: pom.xml [hunk 3]
```diff
@@ -1,23 +1,771 @@
...
+              </pluginExecutions>
+            </lifecycleMappingMetadata>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.openhab.tools.sat</groupId>
+          <artifactId>sat-plugin</artifactId>
+          <version>${sat.version}</version>
+          <configuration>
+            <checkstyleProperties>${basedirRoot}/../../tools/checkstyle/ruleset.properties</checkstyleProperties>
```
Comment (J-N-K) [inline pom.xml]: Is this still true? `${basedirRoot}` should be `/` if you move the reactor pom there

File: pom.xml [hunk 4]
```diff
@@ -1,23 +1,771 @@
...
-  -->
+    <profile>
+      <id>skip-itest</id>
+      <activation>
+        <property>
+          <name>!skipTests</name>
+        </property>
+      </activation>
+      <modules>
+        <module>../../itests</module>
```
Comment (J-N-K) [inline pom.xml]: is this still true? I would guess you could omit `../../` now

File: pom.xml (hunk 1 above)
Comment (davidgraeff) [reply pom.xml]: I moved the poms/bnd/pom.xml file to root and it was defined like this by Markus I guess.
If the guys over at OH1-addons don't start migrating they will not have a working IDE setup anymore in a few days, so we don't actually have OH1 vs OH2 anymore and could also just say "OpenHAB Add-Ons" here.

File: pom.xml (hunk 2 above)
Comment (davidgraeff) [reply pom.xml]: That depends if you take the current Eclipse ESH.xml as formatting reference.

**Comment by maggu2810** [comment]
Hm, why not changing the group IDs to fit the directory layout?

**Comment by davidgraeff** [comment]
Too many entanglements in other repos. I wanted group and artifact ids to be untouched in this PR.
